src/dialogsettings.cpp

```diff
@@ -84,6 +80,21 @@ DialogSettings::DialogSettings( QWidget *parent)
     spinThunderbirdStartDelay->setValue( settings->mLaunchThunderbirdDelay );
     boxShowUnreadCount->setChecked( settings->mShowUnreadEmailCount );
 
+    // Form the proper command-line (with escaped arguments if they contain spaces
+    QString tbcmdline;
+    for ( QString a : settings->mThunderbirdCmdLine )
+    {
+        if ( !tbcmdline.isEmpty() )
+            tbcmdline += ' ';
+
+        if ( a.contains( ' ') )
```

Comment:
Don't add quotes if they are already present, otherwise you get `""%ProgramFiles(x86)%\Mozilla Thunderbird\thunderbird.exe""` for the default path on Windows.